rules/endpoint/path-must-specify-tags.spec.ts

```diff
@@ -60,4 +65,115 @@ describe("path-must-specify-tags", () => {
         },
       },
     });
+
+const getComponentTestSpec = (tags?: string[], path = "/api/some/path") =>
+  JSON.stringify({
```

Comment:
I don't think it's neccessary to have the full spec for this unit test. Only a path and a component should be neccessary to reproduce the issue.